src/sections/Cloud-Native-Catalog/patterns.js

```diff
@@ -0,0 +1,321 @@
+import React from "react";
+import styled from "styled-components";
+import Mutual_tls from "../../assets/images/meshmap/catalog-card-images/mutual-tls.svg";
+import Retries from "../../assets/images/meshmap/catalog-card-images/retries.svg";
+import Traces from "../../assets/images/meshmap/catalog-card-images/traces.svg";
+import Denial from "../../assets/images/meshmap/catalog-card-images/denial.svg";
+import Correlate_event from "../../assets/images/meshmap/catalog-card-images/correlate-event.svg";
+import Only_wagent from "../../assets/images/meshmap/catalog-card-images/only-wagent.svg";
+import Node_agent from "../../assets/images/meshmap/catalog-card-images/node-agent.svg";
+import Single_tenant from "../../assets/images/meshmap/catalog-card-images/single-tenant.svg";
+import Pre_provison from "../../assets/images/meshmap/catalog-card-images/pre-provision.svg";
+import Circuit_breaker from "../../assets/images/meshmap/catalog-card-images/circuit-breaker.svg";
+import Retry_deadline from "../../assets/images/meshmap/catalog-card-images/retry-deadline.svg";
+import Singleton from "../../assets/images/meshmap/catalog-card-images/singleton.svg";
+import Jwt_transformer from "../../assets/images/meshmap/catalog-card-images/jwt.svg";
+import Multicluster from "../../assets/images/meshmap/catalog-card-images/multicluster.svg";
+import Http_metrics from "../../assets/images/meshmap/catalog-card-images/http.svg";
+import { Container } from "../../reusecore/Layout";
+
+const CatalogWrapper = styled.div`
+  margin: 0;
+  min-height: fit-content;
+  border-width: 2px 2px 2px 2px;
+
+  .catalog-container {
+    padding: 0;
+  }
+
+  .catalog-wrapper {
+    box-shadow: 0px 6px 5px 0px rgba(0, 0, 0, 0.25);
+    padding: 4rem;
+    background-color: ${(props) => props.theme.secondaryColor};
+    background: linear-gradient(
+      180deg,
+      hsla(173, 100%, 35%, 1) 7%,
+      hsla(0, 0%, 0%, 1) 90%
+    );
+  }
+
+  h3.containerCaption {
+    color: ${(props) => props.theme.saffronColor};
+    padding-top: 0.25rem;
+    margin-bottom: -0.5rem;
+    font-size: 1.44rem;
+    font-weight: normal;
+    text-align: center;
+    span {
+      color: ${(props) => props.theme.saffronColor};
+    }
+  }
+  .vertical-line {
+    position: relative;
+    margin: auto;
+    height: 20rem;
+    width: 0px;
+    border: 1.5px solid ${(props) => props.theme.green00B39FToGreyB3B3B3};
+    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+    @media (max-width: 468px) {
+      display: none;
+    }
+  }
+
+  .container {
+    margin: 4rem auto;
+    width: 120px;
+    height: 150px;
+    position: relative;
+    perspective: 1000px;
+  }
+
+  #carousel {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    transform-style: preserve-3d;
+    animation: swirl 40s steps(10000, end) infinite;
+  }
```

Comment:
```suggestion
  }
  
```
line break consistency